package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "action-cli-deploy",
+  "version": "1.0.0",
+  "description": "GitHub Action to deploy an app using Office 365 CLI",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
@anoopt, would you please be able to add build task with the command you've used to generate the dist/index.js file?

I assume it is something like tsc -p tsconfig.json, but it would be nice to have it here documented and ready to be run again in a case of changes.